dbt/include/bigquery/macros/materializations/seed.sql

```diff
@@ -10,8 +10,9 @@
 {% macro bigquery__load_csv_rows(model, agate_table) %}
 
   {%- set column_override = model['config'].get('column_types', {}) -%}
+  {%- set kms_key_name = model['config'].get('kms_key_name', {}) -%}
```

Comment:
I think the data type of `kms_key_name` is `str`. If so, does `{}` make sense as the default here? Is the expected arg datatype on `EncryptionConfiguration` `str` or `dict`?